src/components/bookings.tsx

```diff
@@ -59,7 +63,14 @@ const Trans: React.SFC<TransactionProps> = ({
           {transaction.pedelecName.replace(/_[nN]/g, "")}
           {", "}
           {moment(endDate, undefined, language).from(startDate, true)}
-          {transaction.credited && `, ${menu.REFUNDED}`}
+          {transaction.fees !== 0 && (
+            <>
+              {", "}
+              <span style={transaction.credited ? strikethorughStyle : {}}>
+                {transaction.fees}€
```

Comment:
Oh and please make sure the number is formatted as currency. There is a helper defined here: https://github.com/NeoLegends/velocity-pwa/blob/master/src/util/to-euro.ts